src/main/resources/static/css/style.css

```diff
@@ -2887,20 +2892,8 @@ img.centered-mini {
 		left: calc(50vw - 183px);
 	}
 }
-	
-@media only screen and (min-width: 1380px) {
-	.article {
-		min-width: 580px;
-	}
-	
-	.article-set:after {
-		display: block;
-		content: "  ";
-		flex: 999 999 auto;
-	}
-}
```

Comment:
Previously, on an exact resolution it looked like this:
![Screenshot 2020-07-01 22 51 13](https://user-images.githubusercontent.com/60111772/86296271-b3bccd80-bbf8-11ea-9175-8747f49d2eb3.png)
Also, the nasty `::after` pseudo-element was unneccesary.